client/less/main.less

```diff
@@ -1071,6 +1071,21 @@ code {
       background-color: #666;
     }
   }
+  .hamburger-dropdown {
+    .dropdown .dropdown-menu,
+    .open #nav-Community-dropdown {
```

Comment:
@systimotic So the reason I left it like that was because when I only nested the link hovering section inside of `.hamburger-dropdown`, when the page was resized to smaller widths, part of the styling no longer applied:
![image](https://cloud.githubusercontent.com/assets/18563015/22002526/ad23e876-dc1a-11e6-8be5-f75531e78da8.png)
All the `a` backgrounds should still be the dark gray, so it is inconsistent if left along on page resize. But I hear you on the distraction of the hover, I refactored it all a bit to combine everything with the original lines you pointed out in the OP, and it all seems to work nicely, at any page size, with the exact same amount code.

I did ass some extra code though,  for styling for the `.menu-toggle` button which I left out the first time, and since I was making changes to this PR, I'm just going to add the modal styles for #12558 here too. Let me know what you think when I push it through.